app/locales/zh-tw/translations.js

```diff
@@ -760,7 +758,7 @@ export default {
       "增加{{state}}的數量到{{totalQty}} 對於{{designationCode}}! ({{qty}} 原本{{pkgState}}-物件{{inventory number}})",
     warning_text_without_link:
       "增加{{state}}的數量到{{totalQty}} 對於{{orderCode}}! ({{qty}} 原本{{designatedState}})",
-    designate_to: "指定這組物件到{{orderCode}}",
+    designate_to: "Designate this package to:",
     designate_entire: " 指定全數",
     designate_part: "指定部份"
```

Comment:
`quantity_input: "Quantity to designate"` missing in chinese file.

Reply:
good eye 👍